### PR TITLE
AWSCLOUDWATCHMETRICSTREAM

### PR DESCRIPTION
**Pull request by morecar**
### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an explanation above.

### CODE REVIEW

File: entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+providerAccountName:
+  title:  account
```
Comment (lujop) [inline entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml:2]:
```suggestion
  title: account
```

File: entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+metricUpdate:
+  goldenMetric: metricUpdate
+  title: Metric Updates
+  unit: COUNT
+totalMetricUpdate:
+  goldenMetric: totalMetricUpdate:
```
Comment (lujop) [inline entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml:16]:
```suggestion
  goldenMetric: totalMetricUpdate
```

File: entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  title: Total Metric Updates
+  unit: COUNT
+publishErrorRate:
+  goldenMetric: publishErrorRate:
```
Comment (lujop) [inline entity-types/infra-awscloudwatchmetricstream/summary_metrics.yml:20]:
```suggestion
  goldenMetric: publishErrorRate
```

**Review by lujop — COMMENTED** [review]
Review proposed changes

**Review by albertoolivan — APPROVED** [review]

**Comment by albertoolivan** [comment]
gc build

**Comment by albertoolivan** [comment]
ok jenkins

**Review by naxhh — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Pull request merged**